sympy/polys/tests/test_numberfields.py

```diff
@@ -545,6 +545,7 @@ def test_AlgebraicNumber():
     assert a.is_aliased is True
 
     assert AlgebraicNumber(sqrt(2), []).rep == DMP([], QQ)
+    assert AlgebraicNumber(sqrt(2), (0, 0)).rep == DMP([], QQ)
```

Comment:
Yes. ``DMP.from_sympy_list(sympify(coeffs), 0, dom)`` treats tuples and lists in the same way.